hivemind/proto/p2pd.proto

```diff
@@ -0,0 +1,158 @@
+syntax = "proto2";
```

Comment:
I suggest to make a docstring clarifying that this code comes from the [repo](https://github.com/mhchia/py-libp2p-daemon-bindings), mentioning the license and author name.